_posts/news/2023-01-26-announcing-scalajs-1.13.0.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Announcing Scala.js 1.13.0
+category: news
+tags: [releases]
+permalink: /news/2023/01/26/announcing-scalajs-1.13.0/
+---
+
+
+We are excited to announce the release of Scala.js 1.13.0!
+
+**This release drops support for Scala 2.11.**
+Finally, 5 years after Scala 2.11 was declared end-of-life, we decided to drop support for it in Scala.js.
+We also dropped support for Scala 2.12.1 (Scala 2.12.2+ is still supported, of course).
+
+Other than that, this release brings a number of bug fixes and enhancements.
+Notably:
+
+* Ability to export inner `object`s in Scala classes
```

Comment:
We also support exporting inner classes :) (as factory methods)

Reply:
🤦‍♂️